mxd/docs/File Transfer Azure to Azure.md

```diff
@@ -0,0 +1,416 @@
+# File Transfer: Azure Storage to Azure Storage
+
+## 1. Description
+This tutorial illustrates the process through which a provider participant can transfer a file stored in its Azure Storage to another Azure Storage belonging to a consumer participant.  
+We assume `Alice` as a provider participant and `Bob` as consumer participant.
```

Comment:
We should explain the setup, e.g. that we use a self-contained version of Azure Blob Storage named "Azurite", and that we deploy one Azurite instance for both connectors, due to "a limitation in overriding endpoints".

Reply:
Done